client/.stylelintrc

```diff
@@ -32,6 +32,7 @@
     "scss/at-import-partial-extension-blacklist": ["scss"],
     "scss/at-extend-no-missing-placeholder": true,
     "scss/at-import-no-partial-leading-underscore": true,
+    "scss/dollar-variable-pattern": "^[a-z0-9]+((-[a-z0-9]+))*$",
```

Comment:
Reading it are we forcing a hyphen? What about `$blue`? I might be completely wrong as reading regex isn't exactly natural.

We could also call it something more descriptive i.e. `not-camel-case` or `hyphened-variable`?